udunits2/meta.yaml

```diff
@@ -1,11 +1,11 @@
 package:
     name: udunits2
-    version: 2.2.17
+    version: 2.2.20
```

Comment:
So is this version 2.2.20 of udunits-2? Or is it version 2.2.20 of udunits? :stuck_out_tongue_winking_eye: